spec/ravelin/client_spec.rb

```diff
@@ -12,28 +12,6 @@
 
   let(:client) { described_class.new(api_key: 'abc') }
 
-  shared_context 'event setup and stubbing' do
```

Comment:
I might be missing something, it's not clear to me why these tests should be deleted?